practicaenclases/Practica.py

```diff
@@ -0,0 +1,103 @@
+print "--------------1----------------------"
+
+
+def factorial(num):
+    if num > 0:
+        i = 1
+        Result = 1
+        while i <= num:
+            Result = Result * i
+            i += 1
+        return Result
+
+
+print factorial(0)
+
+print "--------------2----------------------"
+
+
+def is_capicua(num):
+    num = str(num)
+    chain = ""
+    j = len(num) - 1
+    while j >= 0:
+        chain = chain + num[j]
+        j -= 1
+    if chain == num:
+        return "el numero es capicua"
+    else:
+        return "el numero no es capicua"
+
+
+print is_capicua(55)
```

Comment:
remove this, you are using unit test to test your method

Reply:
commit e6f6b4c6f43732aa5fb9731fbf56be44dd717897

Reply:
commit e6f6b4c6f43732aa5fb9731fbf56be44dd717897